meta/FToFConversionInNucleus.m

```diff
@@ -0,0 +1,214 @@
+(* ::Package:: *)
```

Comment:
Why should this file be part of this pull request? It seems to me, it should be in a separate pull request.